addon/components/freestyle-subsection.js

```diff
@@ -6,7 +6,7 @@ const { computed, inject } = Ember;
 export default Ember.Component.extend({
   layout,
   classNames: ['FreestyleSubsection'],
-
```

Comment:
I like to follow [SUIT CSS naming conventions](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md), which have opinions on state modifier selectors like this... basically `is-stateName`.

So how about changing the this to:

```js
classNameBindings: ['show:is-showing:is-hidden'],
```